book/MCDM Productions; Arcadia Issue 8.json

```
@@ -0,0 +1,3593 @@
+{
+    "$schema": "https://raw.githubusercontent.com/TheGiddyLimit/homebrew/master/_schema-fast/homebrew.json",
+    "_meta": {
+        "sources": [
+            {
+                "json": "Ar8",
+                "abbreviation": "Ar8",
+                "full": "Arcadia Issue 8",
+                "url": "https://shop.mcdmproductions.com/products/arcadia-issue8",
+                "authors": [
+                    "James Intracaso",
+                    "Matt Colville",
+                    "Jessica Marcrum",
+                    "V.J. Harris",
+                    "Bianca Bickford"
+                ],
+                "convertedBy": [
+                    "ForeverAlwaysGM (Billiam94)"
+                ],
+                "version": "10/2021"
+            }
+        ]
+    },
+    "subclass": [
+        {
+            "name": "Hag Patron",
+            "shortName": "Hag",
+            "source": "Ar8",
+            "page": 3,
+            "subclassFeatures": [
+                "Warlock Hag Patron|Warlock||Hag|Ar8|1",
+                "Hag Kind|Warlock||Hag|Ar8|1",
+                "Expanded Spell List|Warlock||Hag|Ar8|1",
+                "Bonus Cantrip|Warlock||Hag|Ar8|1",
+                "Horrifying Appearance|Warlock||Hag|Ar8|1",
+                "Auntie's Travel Guide|Warlock||Hag|Ar8|6",
+                "Wicked Claws|Warlock||Hag|Ar8|10",
+                "Grandmother's Wrath|Warlock||Hag|Ar8|14"
+            ],
+            "className": "Warlock",
+            "classSource": "PHB",
+            "subclassSpells": [
+                "vicious mockery"
+            ],
+            "additionalSpells": [
+                {
+                    "known": {
+                        "0": [
+                            "vicious mockery"
```

Tag cantrips in `additionalSpells` like that
```suggestion
                            "vicious mockery#c"
```